server/form-pages/apply/risks-and-needs/health-needs/mentalHealth.ts

```diff
@@ -63,7 +82,20 @@ export default class MentalHealth implements TaskListPage {
   }
 
   response() {
-    const response = {}
+    const response = {
+      [this.questions.hasMentalHealthNeeds.question]: sentenceCase(this.body.hasMentalHealthNeeds),
+      [this.questions.hasMentalHealthNeeds.needsDetail.question]: this.body.needsDetail,
+
+      [this.questions.isEngagedWithCommunity.question]: sentenceCase(this.body.isEngagedWithCommunity),
+      [this.questions.isEngagedWithCommunity.servicesDetail.question]: this.body.servicesDetail,
+
+      [this.questions.hasPrescribedMedication.question]: sentenceCase(this.body.hasPrescribedMedication),
```

Comment:
I like using this `sentenceCase` and putting all the question data in one object, rather than the `options` objects we have on other pages - it was really annoying me but didn't feel like I had the headspace to look at it! Nice one!

Reply:
yes I think that `options()` function can get pretty complex -- and at present it doesn't really need to be!

Reply:
yes I think that `options()` function can get pretty complex -- and at present it doesn't really need to be!